### ISSUE TITLE
DSC in FDS 6.8.0 seems off compared to FDS 6.7.9, using TGA_ANALYSIS

### ISSUE THREAD

**Issue by TristanHehnen** [firemodels/fds#11932]
Hello everybody!

I realised a discrepancy in the DSC data, produced from `TGA_ANALYSIS` compared between 6.7.9 and 6.8.0.

![grafik](https://github.com/firemodels/fds/assets/34650649/2c93b9da-2f69-40c4-983e-72603ee050df)

The data series labelled "FDS6.8.0, Vali." is produced from the [NIST Polymer case](https://github.com/firemodels/fds/blob/master/Validation/NIST_Polymers/FDS_Input_Files/NIST_TGA_10K.fds), it is below the orange one.

The other two are using TGA_ANALYSIS, like this:

```
&SURF ID                      = 'PMMA TGA'
      TGA_ANALYSIS            = .TRUE.,
      TGA_HEATING_RATE        = 10.1,
      TGA_FINAL_TEMPERATURE   = 600.0,
      RGB                     = 200,100,0,
      BACKING                 = 'EXPOSED',
      MATL_ID(1,1:1)          = 'BLACK PMMA',
      MATL_MASS_FRACTION(1,1) = 1.0 /

``` 

FDS 6.8.0 yields a very similar response as the validation case, using TGA_ANALYSIS, see "FDS6.8.0, TGA_ANALYSIS".
However, things look very different for FDS 6.7.9, see "FDS6.7.9, TGA_ANALYSIS".

The individual heating rates are not too far off, though (don't mind the +0.1 for the TGA_ANALYSIS cases).

![grafik](https://github.com/firemodels/fds/assets/34650649/fdc741b7-9775-4707-8cfa-f93b56262ece)

I wonder if something fundamental has  changed or if this is an issue only with the "DSC" from `TGA_ANALYSIS`.

I've used these specific versions:
`FDS6.7.9-0-gec52dee-HEAD`
`FDS-6.8.0-328-g6a323c5-master`

**Comment by drjfloyd** [comment]
This case was added April this year for the 6.8 release, it wasn't part of the validation suite for 6.7.9. There has been work on the solid phase model since the 6.7.9 release. The 6.8.0 data looks to be correct.  If you look at one of the measured DSC curves (say this one: https://github.com/MaCFP/matl-db/blob/master/PMMA/Calibration_Data/NIST/NIST_DSC_N2_10K_5.csv), the peak is ~3 W/g.

**Comment by TristanHehnen** [comment]
Yes, I've just checked with our MaCFP-3 parameter set in a simplified cone calorimeter setup. It seems that the difference is related to the DSC part of TGA_ANALYSIS, because the parameter set itself behaves basically identical for both versions:

![grafik](https://github.com/firemodels/fds/assets/34650649/e2157085-2416-4fbf-bf0e-fefc3ccde1c5)

**Comment by TristanHehnen** [comment]
To the experiment comparison:
I've seen that the simulation response from the new FDS validation case fits to the 10K/min DSC data from MaCFP (FSRI). I've built our parameter set on it, using FDS 6.7.9. with `TGA_ANALYSIS`. I wanted to compare it with the FDS validation response, so I adjusted the surface definition as described above. 
Then I saw that it is off, as shown above. Using the FDS validation case as presented with FDS6.8.0 it fits well:

![grafik](https://github.com/firemodels/fds/assets/34650649/d5b1102b-148c-489c-8f83-f2273db17e57)

Thus, it seems related to the `TGA_ANALYSIS`.
